app/src/main/java/com/gathering/android/signUp/model/ResponseState.kt

```diff
@@ -0,0 +1,8 @@
+package com.gathering.android.signUp.model
+
+sealed interface ResponseState {
+
+    class Success(val user: User) : ResponseState
```

Comment:
Good idea to have a proper result from the model layer to view model 💪

Reply:
thanks